ui/src/layouts/Baseline.vue

```diff
@@ -209,6 +210,15 @@ export default {
             } else {
                 this.drawer = !this.drawer
             }
+        },
+        selectPage (page) {
```

Comment:
A key practice I've adopted is prefixing event listener method names with 'on'. This makes their role clear and enhances code readability.